R/pkg/tests/fulltests/test_sparkSQL.R

```diff
@@ -1739,6 +1748,13 @@ test_that("string operators", {
     collect(select(df5, repeat_string(df5$a, -1)))[1, 1],
     ""
   )
+
+  l6 <- list(list(a = "abc"))
+  df6 <- createDataFrame(l6)
+  expect_equal(
+    collect(select(df6, reverse(df6$a)))[1, 1],
+    "cba"
+  )
```

Comment:
Let's make this inlined while we are here. I would also put this test around 1505L above.